hydra-web/src/js/app.js

```diff
@@ -119,17 +129,17 @@ function(
             var urlPath;
             if (tokenName == "token") {
                 urlPath = "/authentication/login";
-            } else if (tokenName == "sudo") {
-              urlPath = "/authentication/sudo";
             } else {
                 alertify.error("Unknown token name " + tokenName);
                 return;
             }
             var username = $.trim(usernameInput.value);
             var password = passwordInput.value;
+            passwordCache = passwordInput.value;
```

Comment:
I would **HIGHLY DISCOURAGE** any attempt to save ldap passwords on the browser side. I recommend that we completely change our approach to something that does not involve the caching of passwords on the browser.